tests/unit/copyTemplate.test.js

```diff
@@ -0,0 +1,148 @@
+const { expect } = require('chai')
+const sinon = require('sinon')
+const fs = require('fs-extra')
+const path = require('path')
+const git = require('isomorphic-git')
+const CreateAioLibCommand = require('../../src/commands/create')
+
+const chai = require('chai')
+const chaiAsPromised = require('chai-as-promised')
+
+chai.use(chaiAsPromised)
+
+describe('CreateAioLibCommand', () => {
+  let sandbox
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox()
+  })
+
+  afterEach(() => {
+    sandbox.restore()
+  })
+
+  describe('copyTemplate', () => {
```

Comment:
the approach is ok, but I was looking for the Command being a unit: tests that will just test the command + flags + args combinations and what is expected. This will in turn test the internal functions indirectly, which are not really exposed to anyone for use. 

See https://github.com/adobe/aio-cli-plugin-app/blob/master/test/commands/app/init.test.js

Reply:
I'll enable the Github Actions workflow to be run, and we can see how the tests fare plus the level of coverage, see if it is enough.